common/src/View/EntityAttributeGridTable.cpp

```diff
@@ -511,6 +511,57 @@ namespace TrenchBroom {
             m_showDefaultRows = showDefaultRows;
             update();
         }
+        
+        static StringSet allSortedAttributeNames(MapDocumentSPtr document) {
+            StringSet keySet;
+            const Model::AttributableNodeList nodes = document->allAttributableNodes();
```

Comment:
As below, you should use the World's attribute index to get all attribute names. But be aware of numbered attributes, too (target1, ...).